pom.xml

```diff
@@ -792,7 +792,7 @@
         <spark.test.version>${spark.version}</spark.test.version>
         <spark.version.classifier>spark${buildver}</spark.version.classifier>
         <cuda.version>cuda11</cuda.version>
-        <cudf.version>22.06.0-SNAPSHOT</cudf.version>
+        <spark-rapids-jni.version>22.06.0-SNAPSHOT</spark-rapids-jni.version>
```

Comment:
we need to update the deploy script to push the new jar with classifier. I assume we have to push both classifier and non-classifier version.  @pxLi  perhaps has more insight.

Reply:
Yeah, currently we have both in our nightly deployment
https://oss.sonatype.org/content/repositories/snapshots/com/nvidia/spark-rapids-jni/22.06.0-SNAPSHOT/

Reply:
also I saw json has already update default jar to classifier one, so we may just need to confirm it works w/ all our deploy scripts https://github.com/NVIDIA/spark-rapids/pull/5249/files#diff-2681b51a40a6a26d3a6f7f839cd810422226193ef273ea02d1b0b93c3a87cfa6R220-R222